app/controllers/v0/sessions_controller.rb

```diff
@@ -84,7 +84,7 @@ def saml_user
 
     def create_saml_user
       user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1?
+      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? # || user.gender.nil?
```

Comment:
@omgitsbillryan did some more testing and sometimes MVI would find a user without middle name so yes I think I think we should still try, worst case they are not found and the user continues without a va_profile.